### PR TITLE
Feat/3d

### PR DESCRIPTION
**Pull request by lniehaus**
Enable Storage urls in DEBUG Mode

### CODE REVIEW

**Comment by lniehaus** [comment]
In Addition the "make run-frontend" command is added

**Comment by lniehaus** [comment]
@jarkenau you will need these changes when you want to complete this Issue https://github.com/naturerobots/HSOS-SEP-PlantMap-2022/issues/92 and make the files avaiable in the storage/media folder

**Comment by lniehaus** [comment]
I have no experience in what will happen when this pull request and the Feat/tokenauth https://github.com/naturerobots/HSOS-SEP-PlantMap-2022/pull/129 pull request will be merged, since we both change the same files

File: django/config/urls.py [hunk 1]
```diff
@@ -20,3 +20,21 @@
     path('admin/', admin.site.urls),
     path('', include('restapi.urls')),
 ]
+
+# http://localhost:8000/media/pointclouds/ply/e1ef73b1258b475a996d2b72924c27ac/0bf37a0851b7402d88674e153f58e6f8.ply
+
+# #This is only needed when the DEBUG Environment variable is used
```
Comment (jarkenau) [inline django/config/urls.py:26]:
```suggestion
# This is only needed when the DEBUG Environment variable is used
```

**Comment by lucaslootalot** [comment]
@lniehaus The first to merge the PR will have no problems, the second will have to resolve the conflicts manually. This should not be a problem with one file.

File: django/restapi/tasks.py [hunk 2]
```diff
@@ -107,7 +108,11 @@ def save_ply(pcloud: PointCloud2, puuid: String, uuid: String):
...
 
         # https://docs.djangoproject.com/en/4.0/topics/files/
+
         file_path = 'storage/media/pointclouds/ply/' + puuid + '/' + uuid + '.ply'
+        # file_path = settings.MEDIA_ROOT + 'pointclouds/ply/' + puuid + '/' + uuid + '.ply'
+
+        logger.info("file_path " + str(file_path))
 
         if not default_storage.exists(file_path):
             # file_content = ContentFile(header.encode('utf-8') + '\r\n'.encode('utf-8') + pcloud.data)
```
Comment (jarkenau) [inline django/restapi/tasks.py:118]: Could be removed I guess :thinking: 
```suggestion
```

**Review by Anafabula — APPROVED** [review]

File: django/restapi/tasks.py (hunk 2 above)
Comment (lniehaus) [reply django/restapi/tasks.py:118]: that is correct

File: django/config/urls.py (hunk 1 above)
Comment (lniehaus) [reply django/config/urls.py:26]: you are right, thanks

**Review by jarkenau — APPROVED** [review]

**Pull request merged**